common/scripts/kind_provisioner.sh

```diff
@@ -155,6 +155,9 @@ EOF
   if [[ -n ${METRICS_SERVER_CONFIG_DIR} ]]; then
     kubectl apply -f "${METRICS_SERVER_CONFIG_DIR}"
   fi
+
+  # Install Metallb 
+  install_metallb ""
```

Comment:
slight nit: Do we need to pass `""`? Maybe we can make sure it's set in the `install_metallb` func with `if -z`.